_data/accessibility-tests/process-list.yml

```diff
@@ -0,0 +1,41 @@
+title: Process list
+component_status: pass
+test_items:
+# General tests
+  - summary: Headings are structured in a logical order.
+    summary_additional: When you interact with a process list, you understand headings move in a logical order. For example, headings move from h2 to h3.
+    test_status: conditional
+    test_type: general
+    version_tested: 1.3.1
+    wcag_criterion: 3.10.0
```

Comment:
**issue:** Lil data mixup
```suggestion
    version_tested: 3.10.0
    wcag_criterion: 1.3.1
```

Reply:
Resolved in [73ddb50](https://github.com/uswds/uswds-site/pull/3007/commits/73ddb50d7c1d23da0cfc280840136c43b0245c75)